pkg/prometheus/statefulset_test.go

```diff
@@ -93,6 +93,6 @@ func TestStartupProbeTimeoutSeconds(t *testing.T) {
 
 		require.Equal(t, test.expectedStartupPeriodSeconds, startupPeriodSeconds)
 		require.Equal(t, test.expectedStartupFailureThreshold, startupFailureThreshold)
-		require.Equal(t, test.expectedStartupPeriodSeconds*test.expectedStartupFailureThreshold, test.expectedMaxStartupDuration)
+		require.Equal(t, test.expectedMaxStartupDuration, startupPeriodSeconds*startupFailureThreshold)
```

Comment:
@nicolastakashi @simonpasquier 

This linter caught a silly mistake made in #6137 :P